Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -37,7 +37,6 @@
 #include "Common/Version.h"
 #include "Common/WindowSystemInfo.h"
 
-#include "Core/AchievementManager.h"
```

Comment:
Did a rebase go wrong or something? I don't think this is supposed to be part of the diff